packages/solhint-config-web3studio/index.js

```diff
@@ -1,39 +1,9 @@
 module.exports = {
   rules: {
-    'imports-on-top': 'error',
-    'variable-declarations': 'error',
-    'array-declarations': 'error',
-    'no-unused-vars': 'error',
-    'value-in-payable': 'error',
-    'linebreak-style': 'error',
-    quotes: ['error', 'double'],
-    indentation: ['error', 2],
-    'no-empty-blocks': 'error',
-    whitespace: 'error',
-    'deprecated-suicide': 'error',
-    'pragma-on-top': 'error',
-    'function-whitespace': 'error',
-    'semicolon-whitespace': 'error',
-    'comma-whitespace': 'error',
-    'conditionals-whitespace': 'error',
-    'operator-whitespace': 'error',
-    emit: 'error',
-    'no-constant': 'error',
-    'max-len': 'error',
-    'error-reason': 'error',
-    'visibility-first': 'error',
-    lbrace: 'error',
-    mixedcase: 'error',
-    camelcase: 'error',
-    uppercase: 'error',
-    'no-experimental': 'error',
-
-    'blank-lines': 'off',
-    'arg-overflow': 'off',
-    'function-order': 'off',
-
-    // Disable deprecated rules
-    'double-quotes': 'off',
-    'no-with': 'off'
+    // Security
+    'compiler-fixed': 'error',
+    'mark-callable-contracts': 'off',
+    // Style
+    indent: ['error', 2]
```

Comment:
These are the rules I needed to get bootleg running clean. Likely more will need updating in the future as styles become evident.